test-basic/documents-quick.js

```diff
@@ -167,3 +167,67 @@ describe('quick path', function(){
     .catch(done);
   });
 });
+
+describe('test connection', function(){
+ var assert = require('assert');
+    it('should test the connection', function(done){
+     	var flag = false;
+     db.testConnection()
+      .result(function(response) {
+        flag = true;
+     }).then(function(){
+        assert(flag == true);
+        done();
+    }).catch(done);
+   });
+    it('should give 403 when user is not authorized', function(done){
+    var config = {host:testconfig.restReaderConnection.host, user:'testUser', password: 'test', port:testconfig.restReaderConnection.port, 
+        authType:testconfig.restReaderConnection.authType};
+    var db1 = marklogic.createDatabaseClient(config);
+    var assert = require('assert');
+    var flag = false; 
+    var content = {connected: false,
+       httpStatusCode:  403,
+       httpStatusMessage: 'Forbidden'
+       };
+    var respContent = null;
+     db1.testConnection()
+      .result(function(response) {
+        flag = true;
+        respContent = response.content[0];
```

Comment:
Instead of having to get a content subproperty, would it be better to have the properties directly on the response. 

Same comment on the subsequent test.